tests/Browser/views/layouts/app.blade.php

```diff
@@ -2,23 +2,23 @@
 <head>
     <meta name="csrf-token" content="{{ csrf_token() }}">
     <script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.8.2/dist/alpine.min.js" defer></script>
-    @livewireStyles
```

Comment:
Why this is removed?

Reply:
There was @livewireStyles and livewire:styles 
I thought it was a mistake. It is not so?